executor/benchmark_test.go

```diff
@@ -1452,6 +1452,227 @@ type mergeJoinTestCase struct {
 	childrenUsedSchema [][]bool
 }
 
+func prepare4MergeJoinWithSort(tc *mergeJoinTestCase, innerDS, outerDS *mockDataSource) *MergeJoinExec {
```

Comment:
Can we merge this function with `prepare4ShuffleMergeJoinWithSort`?

Reply:
Of course, already merged.